src/modules/device.json

```diff
@@ -97,6 +97,37 @@
                     }
                 }
             ]
+        },
+        {
+            "name": "developerMode",
+            "tags": [
+                {
+                    "name": "property",
+                    "x-manages": [
+                        "xrn:firebolt:capability:settings:device"
```

Comment:
We want a specific capability for developer mode, imo.

Propose: `xrn:firebolt:capability:security:developer-mode`